contracts/deploy.sh

```diff
@@ -25,15 +25,23 @@ get_rpc_and_verifier() {
   if [[ $script_name == *"L1"* ]]; then
     rpc_url="$T1_L1_RPC"
     verifier="etherscan"
-    verifier_url="https://api-sepolia.etherscan.io/api"
-    api_key_flag="--etherscan-api-key $ETHERSCAN_API_KEY" # Only needed for Etherscan
+    #verifier_url="https://api-sepolia.etherscan.io/api"
+    #api_key_flag="--etherscan-api-key $ETHERSCAN_API_KEY"
+    verifier_url="https://api-sepolia.basescan.org/api"
+    api_key_flag="--etherscan-api-key $BASECAN_API_KEY"
```

Comment:
why? Seems like we are demarcating L1 and PR1 but this makes it seem that any script with L1 in the name is destined for PR1